src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -529,4 +529,21 @@
   <data name="PreferIsKindFix" xml:space="preserve">
     <value>Use 'IsKind' instead of 'Kind'</value>
   </data>
+  <data name="MissingSourceGeneratorAttributeDescription" xml:space="preserve">
+    <value>'{0}' implements 'ISourceGenerator' but does not have the 'Generator' attribute.</value>
+  </data>
+  <data name="MissingSourceGeneratorAttributeMessage" xml:space="preserve">
+    <value>Missing 'Generator' attribute</value>
+  </data>
+  <data name="MissingSourceGeneratorAttributeTitle" xml:space="preserve">
+    <value>Missing 'Generator' Attribute</value>
+  </data>
+  <data name="AddGeneratorAttribute_1" xml:space="preserve">
```

Comment:
We are not really consistent for naming of the code fix titles but I would avoid `_X`.

Reply:
What is the way to distinguish between this and `AddGeneratorAttribute_2`?

Reply:
Maybe `_OneLanguage` and `_TwoLanguages`? Otherwise you could use the naming style from Roslyn here.